profile/README.md

```diff
@@ -79,10 +77,17 @@ Popular application forks with NeoFS support added:
 
 # NeoGo
 
-Take a look at [neo-go](https://github.com/nspcc-dev/neo-go) repository, it's
-all there. [ansible-neogo-n3](https://github.com/nspcc-dev/ansible-neogo-n3)
-can help you running it.
+Take a look at [neo-go](https://github.com/nspcc-dev/neo-go) repository, it's all there.
```

Comment:
I'd say that this one can be left as is as well.

Reply:
The same here.

Reply:
Ditto.